ontologies/CALM_single_MS.md

```diff
@@ -0,0 +1,24 @@
+
+# JSON examples: a free-standing manuscript described in CALM
+
+Comment, lines 8-13
+CALM contains both RefNo (to build the hierarchy) and AltRefNo (to show to the user).
+Both will presumably be needed.  At present each maps simply to identifier (=sourceIdentifier in the .ttl files).  
+How to distinguish?
+Also occurs with Item, below
+
+Comment, lines 47-52
+CALM contains both RefNo (to build the hierarchy) and AltRefNo (to show to the user).
+Both will presumably be needed.  At present each maps simply to identifier (=sourceIdentifier in the .ttl files).  
+How to distinguish?
```

Comment:
In the CALM web front end, only AltRefNo is visible - RefNo is masked completely - and it's called Reference.  In Encore, alas, we see both, AltRefNo as part of the shelf mark and RefNo tagged as Reference - the field that we mapped it into as part of the MARC harvesting was set to be visible because of some other uses of it in other systems.